phinx.json

```diff
@@ -0,0 +1,36 @@
+{
+  "paths": {
+    "migrations": "%%PHINX_CONFIG_DIR%%/migrations"
```

Comment:
Yeah I would prefer that too. 

Hmm, well doing that with JSON is no issue at all with json_encode. Converting to yaml, however requires either the yaml php extension, writing a yaml generator or introducing one as a dependency. Would it be worth introducing these dependencies/complexities over simply ensuring all three are unit tested in the appropriate place?